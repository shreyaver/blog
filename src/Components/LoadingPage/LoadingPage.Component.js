import React from 'react';
import Header from '../Header/Header.Component';
import Footer from '../Footer/Footer.Component';
import './LoadingPage.Component.css';

const LoadingPage = (props) => {
  return (
    <div>
      <Header />
      <div className = "Loading-page"> 
      {/* <i class="icomoon icon-confused"></i> */}
      Loading...
      </div>
      <Footer />
    </div>
  );
}

export default LoadingPage;