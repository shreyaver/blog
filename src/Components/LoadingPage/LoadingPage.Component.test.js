import React from 'react';
import renderer from 'react-test-renderer';
import LoadingPage from './LoadingPage.Component.js';

describe('LoadingPage', () => {
  it('renders without crashing', () => {
    const loadingPage = renderer.create(<LoadingPage />).toJSON();
    expect(loadingPage).toMatchSnapshot();
  });
});
