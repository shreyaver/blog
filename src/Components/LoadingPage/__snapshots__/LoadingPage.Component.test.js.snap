// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage renders without crashing 1`] = `
<div>
  <header>
    <div
      className="Content"
    >
      <div>
        <h1>
          The Artifact
        </h1>
        <h2>
          Culture & Art blog
        </h2>
      </div>
      <nav>
        <div
          className="navBar"
        >
          <a
            href="/"
          >
            Blog
          </a>
          <a
            href="#"
          >
            About
          </a>
          <a
            href="#"
          >
            Contact
          </a>
        </div>
      </nav>
    </div>
  </header>
  <div
    className="Loading-page"
  >
    Loading...
  </div>
  <footer>
    <div
      className="footerContent"
    >
      <small>
        Â©artifact.com 2019
      </small>
    </div>
  </footer>
</div>
`;
